.github/workflows/_mac-test-mps.yml

```diff
@@ -96,6 +96,19 @@ jobs:
           # TODO(https://github.com/pytorch/pytorch/issues/79293)
 
           ${CONDA_RUN} python3 test/test_modules.py -k mps --verbose
+      
+      - name: Run MPS Test NN
+         id: test_2
```

Comment:
Can you change the id to something else (e.g test_3)? Having the same test id as other tests (here test modules uses the same) will stall the CI